.github/workflows/test.yml

```diff
@@ -1,6 +1,20 @@
 on: [push, pull_request]
 name: Test
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: "./go.mod"
+      - name: Lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
```

Comment:
Not a fan of this one, let's do go fmt, go vet, and staticcheck.

Reply:
Okay. Done. Thanks!